Disable merge commits for jdt-incubator

mickaelistria opened the pull request:
Merge commit would make further attempt to merge it in upstream JDT harder. Let's disable those.

mickaelistria committed:
Disable merge commits for jdt-incubator

Merge commit would make further attempt to merge it in upstream JDT harder. Let's disable those.

Signed-off-by: Mickael Istria <mistria@redhat.com>

mickaelistria commented:
@fbricon @rgrunber @gorkem @testforstephen I think this requires a project lead approval. Can you please review?

netomi commented on otterdog/eclipse-jdtls.jsonnet:
```diff
@@ -43,6 +43,7 @@ orgs.newOrg('eclipse-jdtls') {
       secret_scanning: "disabled",
       secret_scanning_push_protection: "disabled",
       web_commit_signoff_required: false,
+      allow_merge_commit: false,
```
the property is already define a few lines above, if you just remove the line 37 it should be what you want as the default value for allow_merge_commit is already false.

mickaelistria replied:
Thanks.

mickaelistria committed:
Fix disable merge commits + enable PR updates

PR updates make rebase/review easier without necessarily involving the submitter for a rebase.

Signed-off-by: Mickael Istria <mistria@redhat.com>

netomi commented:
changes are live.

mickaelistria commented:
Thank you!
